programs/axelar-solana-governance-v2/src/instructions/execute_operator_proposal.rs

```diff
@@ -12,6 +12,7 @@ pub struct ExecuteOperatorProposal<'info> {
     pub system_program: Program<'info, System>,
 
     #[account(
+        mut,
```

Comment:
@MakisChristou we needed the `mut` here but actually the tests were passing since we were already providing the mutable account. let's switch to using the pre-generated Anchor accounts structs to be able to catch these bugs more easily.